_rules/block-collapsible-3e12e1.md

```diff
@@ -0,0 +1,489 @@
+		<button onclick="toggleAriaHidden('chapters-navigation')" class="off-screen">Toggle table of content</button>
+
+		<nav id="chapters-navigation">
+			<h1>Content</h1>
+			<ol>
+				<li>Chapter 1</li>
+				<li><a href="/test-assets/bypass-blocks-cf77f2/chapter2.html">Chapter 2</a></li>
+			</ol>
+		</nav>
+
+		<main>
+			<h1>Three Heroes Swear Brotherhood at a Feast in the Peach Garden</h1>
+			<p>
+				Unity succeeds division and division follows unity. One is bound to be replaced by the other after a long span
+				of time.
+			</p>
+		</main>
+	</body>
+</html>
+```
+
+#### Passed Example 6
+
+The navigational [block of repeated content][] can be made non-[visible][] and removed from the accessibility tree by the button at the start of the document.
+
+```html
+<html>
+	<head>
+		<script src="../test-assets/bypass-blocks-cf77f2/toggle-display.js"></script>
+		<title>The Three Kingdoms, Chapter 1</title>
+	</head>
+	<body>
+		<button onclick="hide('chapters-navigation')">Hide table of content</button>
+
+		<nav id="chapters-navigation">
+			<h1>Content</h1>
+			<ol>
+				<li>Chapter 1</li>
+				<li><a href="/test-assets/bypass-blocks-cf77f2/chapter2.html">Chapter 2</a></li>
+			</ol>
+		</nav>
+
+		<main>
+			<h1>Three Heroes Swear Brotherhood at a Feast in the Peach Garden</h1>
+			<p>
+				Unity succeeds division and division follows unity. One is bound to be replaced by the other after a long span
+				of time.
+			</p>
+		</main>
+	</body>
+</html>
+```
+
+#### Passed Example 7
+
+This [document][] has no [block of repeated content][] because there is no other page linked from it.
+
+```html
+<html lang="en">
+	<head>
+		<title>The Three Kingdoms, Chapter 1</title>
+	</head>
+	<body>
+		<aside id="about-book">
+			<h1>About the book</h1>
+			<p>The Romance of the Three Kingdoms is a 14th century historical novel.</p>
+		</aside>
+
+		<main>
+			<h1 id="main">Three Heroes Swear Brotherhood at a Feast in the Peach Garden</h1>
+			<p>
+				Unity succeeds division and division follows unity. One is bound to be replaced by the other after a long span
+				of time.
+			</p>
+		</main>
+	</body>
+</html>
+```
+
+### Failed
+
+#### Failed Example 1
+
+There is no [instrument][] to toggle the navigational [block of repeated content][].
+
+```html
+<html>
+	<head>
+		<title>The Three Kingdoms, Chapter 1</title>
+	</head>
+	<body>
+		<nav id="chapters-navigation">
+			<h1>Content</h1>
+			<ol>
+				<li>Chapter 1</li>
+				<li><a href="/test-assets/bypass-blocks-cf77f2/chapter2.html">Chapter 2</a></li>
+			</ol>
+		</nav>
+
+		<main>
+			<h1>Three Heroes Swear Brotherhood at a Feast in the Peach Garden</h1>
+			<p>
+				Unity succeeds division and division follows unity. One is bound to be replaced by the other after a long span
+				of time.
+			</p>
+		</main>
+	</body>
+</html>
+```
+
+#### Failed Example 2
+
+The [instrument][] to toggle the navigational [block of repeated content][] is not [visible][].
```

Comment:
This example passes the SC and somehow the SCR28 as the instrument is keyboard accessible and hides the navigational content. Consider my comment in the expectation 1 section. Again screen readers don't need any visible instrument to toggle on or off this kind of instrument as long as these are keyboard accessible when getting focus.

Reply:
I disagree that this example passes the SC.
The SC is also beneficial for sigthed users. The Understanding document lists keyboard users, screen magnifier users, and users with cognitive disabilities as benefiting from it.

In this example, a sigthed user doesn't really have a way to bypass the block of repeated content (well, technically it is possible to blindly tab into an out-of-screen link and activate it hoping it does what is needed, but I don't really think this is enough).